pkg/scheduler/plugins/usage/usage.go

```diff
@@ -99,26 +124,26 @@ func (up *usagePlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 		for k, v := range args {
 			key, _ := k.(string)
-			var val float64
-			switch a := v.(type) {
-			case string:
-				val, _ = strconv.ParseFloat(a, 64)
-			case int:
-				val = float64(a)
-			case float64:
-				val = a
-			default:
-				klog.V(4).Infof("The threshold %v is an unknown type", a)
-			}
-			if strings.Contains(key, cpuUsageAvgPrefix) {
-				periodKey := strings.Replace(key, cpuUsageAvgPrefix, "", 1)
-				up.threshold.cpuUsageAvg[periodKey] = val
+			value, _ := v.(int)
+			switch key {
+			case "cpu":
+				up.cpuThresholds = float64(value)
+			case "mem":
+				up.memThresholds = float64(value)
 			}
-			if strings.Contains(key, memUsageAvgPrefix) {
-				periodKey := strings.Replace(key, memUsageAvgPrefix, "", 1)
-				up.threshold.memUsageAvg[periodKey] = val
+		}
+
+		// here is to deal with the case that restart volcano-scheduler and there is no ResourceUsage in cache of volcano-scheduler
+		if source.Period == "" {
```

Comment:
why period == "" means the case volcano restarted?

Reply:
yes,period == "" means   volcano  is just starting

Reply:
why?

Reply:
> why?

I know it now. period is a gloabal variable.